conf/maestro-conf/relayTerm.template.json

```diff
@@ -1,6 +1,6 @@
 {
-	"cloud": "{{ARCH_GW_SERVICES_URL}}/relay-term",
+	"cloud": "http://gateways.local:8080/relay-term",
 	"noValidate": true,
-	"certificate": "{{SSL_CERTS_PATH}}/client.cert.pem",
-	"key": "{{SSL_CERTS_PATH}}/client.key.pem"
+	"certificate": "",
```

Comment:
These removed because we aren't using TLS anymore?
Why are we removing the certificates in the first place if it is already working?

Reply:
Please refer https://jira.arm.com/browse/PELEDGE19-1428 and https://jira.arm.com/browse/IOTCLOUDPR-2624

Reply:
Removing TLS because fp-edge is handling it, not relayTerm